### PR TITLE
AttributeStringSearcher

### PR DESCRIPTION
**Pull request by kirisaki**
# Overview

Added AttributeStringSearcher.

## What I've done

Added draft of AttributeStringSearcher.

## How I tested

Not yet. 

## Which point I want you to review particularly

I need help understanding the correct action of this workflow because I have yet to try FME.
Please check if it matches what you intend.

### CODE REVIEW

File: worker/crates/action-universal/src/attribute/searcher.rs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            Regex::new(&escape(&self.search_in)).map_err(|_| Error::input("Invalid regex"))
+        }?;
+        let output = inputs
+            .ok_or(Error::input("No input dataframe"))?
+            .iter()
+            .map(|(k, v)| {
+                (
+                    k.clone(),
+                    v.as_ref().map(|v| search(v.clone(), &re)),
+                    /*match v {
```
Comment (miseyu) [inline worker/crates/action-universal/src/attribute/searcher.rs:31]: Delete the commented-out sections.

**Comment by miseyu** [comment]
@kirisaki 

Searching is more than retrieving the contents.
I would like the entire column to be retrieved if the data is in the column contents.

search_in: piyo
input
``` json
[
  {"hoge": "fuga piyo", "test": "example"},
  {"piyo": "hogehoge", "test": "example"},
]
```

output
``` json
[
  {"hoge": "fuga piyo", "test": "example"},
]
```

File: worker/crates/action-universal/src/attribute/searcher.rs (hunk 1 above)
Comment (kirisaki) [reply worker/crates/action-universal/src/attribute/searcher.rs:31]: oh, sorry

**Comment by kirisaki** [comment]
I supposed the output is a list of matches, but is it wrong? 

e.g.:
`searchIn: foo`
```input.json
 {
  "sample_data": {
    "bool": true,
    "array": [
      1,
      2,
      3
    ],
    "number": 4,
    "string": "foo is a meta variable.",
    "map": {
      "item1": 1,
      "item2": "foo bar baz",
      "item3": "foo foo foo"
    }
  } 
}
``` 
```output.json
 {
  "sample_data": {
    "bool": true,
    "array": [
      1,
      2,
      3
    ],
    "number": 4,
    "string": ["foo"],
    "map": {
      "item1": 1,
      "item2": ["foo"],
      "item3": ["foo", "foo", "foo"]
    }
  } 
}
```

**Comment by miseyu** [comment]
@kirisaki 
Sorry, yes, it looks fine!

**Comment by kirisaki** [comment]
@miseyu Can you review it again?

**Review by miseyu — APPROVED** [review]
LGTM

**Comment by kirisaki** [comment]
The update was pushed before your approval, so merging is blocked. I'm afraid that recheck PR.

**Review by miseyu — APPROVED** [review]

**Comment by kirisaki** [comment]
Is it blocked yet?

**Review by miseyu — APPROVED** [review]

**Comment by miseyu** [comment]
@kirisaki 
The branch protect setting was incorrect and has been corrected.
Now it is possible to merge.

**Comment by kirisaki** [comment]
Squash or rebase?

**Comment by miseyu** [comment]
@kirisaki 
Please use squash!

**Comment by miseyu** [comment]
I left the merging method as squash only.

**Pull request merged**